### DISCUSSION TITLE
post/kotlin-migration/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#49] [Announcements]
# post/kotlin-migration/

코틀린을 학습하고, 적용하고, 변환해 봤던 이야기와 실제 자바 프로젝트를 변환하는 샘플을 공유합니다.

https://tech.kakaopay.com/post/kotlin-migration/

**Comment by pay-dan-dy** [comment]
장기간에 걸친 자바에서 코틀린으로의 전환 경험을 이렇게 하나의 글로 정리해 쉽게 이해하도록 공유해 주셔서 감사합니다!
